lib/modules/manager/npm/update/locked-dependency/common/parent-version.ts

```diff
@@ -8,15 +8,17 @@ import { api as semver } from '../../../../../versioning/npm';
 
 const pkgCache = new Map<string, Promise<ReleaseResult | null>>();
 
-function getPkgReleasesCached(depName: string): Promise<ReleaseResult | null> {
-  let cachedResult = pkgCache.get(depName);
+function getPkgReleasesCached(
+  packageName: string
+): Promise<ReleaseResult | null> {
+  let cachedResult = pkgCache.get(packageName);
   if (!cachedResult) {
     const lookupConfig: GetPkgReleasesConfig = {
       datasource: 'npm',
-      depName,
+      packageName,
     };
     cachedResult = getPkgReleases(lookupConfig);
```

Comment:
uhhhh, we've a circular reference here. 😱 and no tooling was able to find it 🤔

Reply:
New to this PR or existing?

Reply:
existing, can you open an issue to track?

Reply:
wait, it's inside manager. 🤦‍♂️ so no circle